modules/gathering-data-network-logs.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+
+:_content-type: PROCEDURE
+[id="gathering-data-network-logs_{context}"]
+= Gathering network logs
+
+You can gather network logs on all nodes in a cluster. The script is installed in the `/usr/bin path` of the image.
```

Comment:
Is `gather_network_logs` the script being referred to here, and are they expected to first `cd` to `/usr/bin` to run the command? I see in a nearby procedure in this assembly it uses the following syntax for a different script:

```
$ oc adm must-gather -- /usr/bin/gather_audit_logs
```

It was just a little confusing at first to not know what script it was referring to or how the `/usr/bin` part played in (since it otherwise doesn't appear in the procedure). Maybe provide a little more explanation or adopt the absolute path like the other command?

Reply:
@adellape That's a good catch! I see that `usr/bin` is where all the flag scripts are stored and do not need an explicit mention. Also I confirmed with QE on prepending the `usr/bin` path is not mandatory and it works in both cases. IMO, if both audit logs and network logs are labeled as flags, I wouldn't see a need to attach a path to it. I should be able to right away use it without worrying about where it is located. So in this case, my suggestion would be to remove the path from Audit section and to remove the statement `The script is installed in the `/usr/bin path` of the image.` from this PR. I do not think it's necessary for the user to know where it's located in this context. WDYT?